### DISCUSSION TITLE
Unable to get Multi-APN working

### DISCUSSION THREAD

**Discussion by 0pt1on** [magma/magma#13442] [Q&A]
Hello,

**Environment:**
We have deployed v1.7.0 Orc in Amazon AWS, have a locally deployed AGW on 1.7.0-1648117787-73e61141 at our site.

For this test I am using an AirSpeed 1030 managed externally and configured with AirSpan's Netspan server.

I have a UE connected - manufacturer: Global Telecom, model: Titan 4000.

UE subscriber type is fwa and the site is type lte.

NOTE: Changing nothing in the configs if the eNB is connected to my Raemis Druid EPC everything works fine and the the UE functions without problems, its only when the eNB is connecting to the Magma AGW that things go wonky.

**Problem:**
Regardless of UE config, Multi-APN w/ IP Passthrough or Router Mode w/ a single APN, I can remotely access the radio via the management PDN as expected.

**Additionally, when the UE is in Router Mode with a single PDN everything works fine.**

However, when the UE is in IP-Passthrough with Multi-APN UE traffic stops passing.

AGW logs are quiet when the UE is in Router Mode; however, soon as its in IP-Passthrough with Multi-APN I see the following being spammed.
`
Jul 27 20:49:25 psn-agw-02 sessiond[33105]: I0727 20:49:25.102885 33105 SessionState.cpp:552] Reported rule (allowlist_sid-IMSI901700000032412-hwinternet) not found in IMSI901700000032412-203029, ignoring
`

No other errors are present in the logs or events.

I have run multiple packet captures, read through all the logs with log level DEBUG turned on, but it just appears its not setting up anything for the additional APNs. If I didn't know better I would think Multi-APN is unsupported, but we all know its been supported for quite awhile now.

What would you like to see to debug this problem.

**Comment by 0pt1on** [comment]
Anyone?

**Reply by Radula76** [reply]
You might need an S1 interface trace/pcap.  I suspect that the IP passthrough means that the device actually talking to the internet is not properly tagging the GTP traffic so the AGW doesn't know what to do with it.  We have tested a few different Multi-APN solutions but none of them with Ue in IP-Passthrough.
